football_dictionaries/assignment_2.py

```diff
@@ -1,2 +1,23 @@
+from .assignment_1 import players_as_dictionaries
+
 def players_by_position(squads_list):
-    pass
+    #convert input to dict(players) with players_as_dictionaries
+    #declare dict(result) structure
+    #iterate over dict(players) to populate player data based on player['position']
+    #return dict(result)
+    players = players_as_dictionaries(squads_list)
+    result = {
+        'GK': [],
+        'FW': [],
+        'MF': []
+    }
```

Comment:
These positions are hardcoded. If the data changes, the solution would no longer work.